src/Tizen.Wearable.CircularUI.Forms.Renderer/TwoButtonPopupImplementation.cs

```diff
@@ -182,7 +182,10 @@ public string Title
             set
             {
                 if (_title == value) return;
-                _title = value;
+                _title = value?.Replace("&", "&amp;")
```

Comment:
It should be move to UpdateTitle method. 
becuase it change `Title`
``` c#
 obj.Title = "A&b";
 Console.WritleLine(obj.Title); // A&amp;B 
 obj.Title == "A&B"; // false
```

Reply:
Done